packages/config/lib/helpers/ConfigMergeHelper.ts

```diff
@@ -0,0 +1,49 @@
+import * as _ from "lodash";
+
+import { MissingConfigPropertiesError } from "../errors";
+
+class ConfigMergeHelper {
+  public static checkForMissingConfigProperties(config: object): void {
+    const missingProperties: string[] = [];
+    this.deepForEach(config, (key, val, nestedPath) => {
+      if (val == null) {
+        missingProperties.push(nestedPath);
+      }
+    });
+
+    if (missingProperties.length > 0) {
+      throw new MissingConfigPropertiesError(missingProperties);
+    }
+  }
+
+  public static getProcessEnvironmentConfig(moduleName: string): any {
+    const processEnvironmentConfig = {};
+
+    Object.entries(process.env).forEach(([key, value]: [string, string]) => {
+      const parsedValue: any = this.parseTrueAndFalseToBooleans(value);
+      _.set(processEnvironmentConfig, key, parsedValue);
+    });
+
+    const processEnvironmentConfigOfModule = processEnvironmentConfig[moduleName] || {};
+
+    return processEnvironmentConfigOfModule;
+  }
+
+  private static parseTrueAndFalseToBooleans(value: string): any {
+    const lowerCaseValue = value.toLocaleLowerCase();
+    if (value === "true") return true;
+    if (value === "false") return false;
+    return value;
+  }
```

Comment:
This is a use case where I personally would prefer a short-hand notation.

Reply:
Would be a doubled shorthand notation since there are three possible results: `true`, `false` and `value`

Reply:
I know, yet quite a simple one and therefor arguably easier to read.

Reply:
```js
return (lowerCaseValue === "true") ? true : (lowerCaseValue === "false") ? false : value;
```